### ISSUE TITLE
device.getLauncher().launchYouTube(video) not working

### ISSUE THREAD

**Issue by dayaki** [ConnectSDK/Connect-SDK-Cordova-Plugin#48]
device.getLauncher().launchYouTube(video) indeed launches the YouTube app but always stuck by just showing the youtube logo.

NB: Everything I tried to pick my chromecast device, it always says "Chromecast, DIAL"......

**Comment by anteburazer** [comment]
Hi dayaki

I have the same issue. Did you find a solution for this problem?

Thanks

**Comment by Danyzzz** [comment]
I'm interested...there is a news?

**Comment by dayaki** [comment]
No solution yet.

**Comment by Danyzzz** [comment]
you have found an alternative solution a connectsdk ?

**Comment by anteburazer** [comment]
No, I haven't been trying anything yet. Their support didn't respond on my enquiry so I can't reach them. Please let us know if you find out something.

Thanks

**Comment by Danyzzz** [comment]
There is this question on stackoverflow, someone can be helpful to understand more ?
http://stackoverflow.com/questions/18115563/open-chromecast-youtube-video-from-my-android-app

**Comment by AA12RTYUIOPL** [comment]
I'm facing the same issue. Is there any solution?
